tests/test_integ_builder_osbs.py

```diff
@@ -571,7 +569,47 @@ def test_osbs_builder_with_fetch_artifacts_file_creation(tmpdir, mocker, caplog)
     assert fetch_artifacts[0] == {'md5': '123456',
                                   'target': 'artifact_name', 'url': 'http://random.url/path'}
 
-    assert "Artifact 'artifact_name' added to fetch-artifacts-url.yaml" in caplog.text
+    assert "Artifact 'artifact_name' (as plain) added to fetch-artifacts-url.yaml" in caplog.text
+
+
+def test_osbs_builder_with_fetch_artifacts_url_file_creation(tmpdir, mocker, caplog):
```

Comment:
Do we have tests that check what happens when there are other keys defined for the artifact like `target` or `dest`?

Reply:
I have added more tests and for those with `target` defined. Not sure yet how to handle `dest`